main.go

```diff
@@ -69,7 +74,8 @@ func main() {
 		return nil
 	}
 
-	app.Action = runAction
+	// Default to 'start' is no command is specified
+	app.Action = startCommand.Action
```

Comment:
yep, just checking.  we could probably remove it soon right?  maybe in the docs not have the `runc` example without using `runc start`